version.go

```diff
@@ -1,7 +0,0 @@
-package skopeo
-
-// Version is a version of thils build.
-const Version = "0.1.12-dev"
```

Comment:
> I see, could you instead define an hash or something (even 0x01) in the signature type instead of relying on skopeo global version? are there any downsides?

There already is a format identifier (`critical.type`), but that has a different function.

(I do hope that we will never have to deploy workarounds based on the creator version (`optional.creator`), but I think it is prudent to have that option.)